test/ERC1400Raw.test.js

```diff
@@ -1,4 +1,4 @@
-import { shouldFail } from 'openzeppelin-test-helpers';
+const { shouldFail } = require('openzeppelin-test-helpers');
```

Comment:
Why are we switching to require here?

Reply:
When I run the command:
`yarn truffle test`

I get the following error with 'import'
```
import { shouldFail } from 'openzeppelin-test-helpers';
       ^

SyntaxError: Unexpected token
```
With 'require', all tests pass.